controllers/operands/kubevirt.go

```diff
@@ -294,7 +295,71 @@ func hcWorkloadUpdateStrategyToKv(hcObject *hcov1beta1.HyperConvergedWorkloadUpd
 	return kvObject
 }
 
-func getKVConfig(hc *hcov1beta1.HyperConverged) (*kubevirtcorev1.KubeVirtConfiguration, error) {
+func getRatesFromConfigMap(configMap *corev1.ConfigMap) (*kubevirtcorev1.ReloadableComponentConfiguration, error) {
+
+	qps, ok := configMap.Data["qps"]
+
+	if !ok {
+		return nil, errors.New("qps parameter not found in configmap " + configMap.Name)
+	}
+
+	qpsConverted, err := strconv.ParseFloat(qps, 32)
+
+	if err != nil {
+		return nil, err
```

Comment:
Please use verbose error message. Maybe wrap the error?
```suggestion
		return nil, fmt.Errorf("more verbose error message; %w", err)
```

Reply:
Done. I have also changed `errors.New` by `fmt.Errorf` for the sake of consistency. Please, let me know if this is appropriated.